lib/containers/utils.pm

```diff
@@ -101,6 +101,7 @@ sub registry_url {
 sub basic_container_tests {
     my %args    = @_;
     my $runtime = $args{runtime};
+    record_info "$runtime";
```

Comment:
what does this bring? We already have the runtime name in the test name `engines_and_tools_docker`.

Reply:
none removed and *this change is ready for review*